src/model/log_prob.jl

```diff
@@ -261,12 +261,12 @@ Construct a `Prior` object, contains StarPrior and GalaxyPrior objects
 function construct_prior()
     pp = load_prior()
     star_prior = StarPrior(
-        LogNormal(pp.r_mean[1], pp.r_var[1]),
+        LogNormal(pp.r_μ[1], sqrt(pp.r_σ²[1])),
```

Comment:
I caught a bug here. `LogNormal` is parameterized in terms of sigma, not sigma^2.